clients/unshaded/src/main/java/tachyon/client/lineage/AbstractLineageClient.java

```diff
@@ -49,10 +51,10 @@ public long createLineage(List<TachyonURI> inputFiles, List<TachyonURI> outputFi
           throws FileDoesNotExistException, TachyonException, IOException {
     // TODO(yupeng): relax this to support other type of jobs
     Preconditions.checkState(job instanceof CommandLineJob, "only command line job supported");
-
     LineageMasterClient masterClient = mContext.acquireMasterClient();
     try {
-      long lineageId = masterClient.createLineage(inputFiles, outputFiles, (CommandLineJob) job);
+      long lineageId = masterClient.createLineage(stripURIList(inputFiles),
```

Comment:
Alternatively, we could still pass `inputFiles` and `outputFiles` here, but have the list transform to strings inside `LineageMasterClient`. @yupeng9 , which approach makes more sense to you?